src/components/Card/Card.jsx

```diff
@@ -119,6 +122,7 @@ Card.propTypes = {
   type: PropTypes.oneOf([cardTypes.REGULAR, cardTypes.CONDENSED]),
   onClick: PropTypes.func,
   borderType: PropTypes.string,
+  borderColorRed: PropTypes.bool,
```

Comment:
prop name should be `borderRed` and the prop type can be changed to `PropTypes.oneOf([borderColors.RED])` instead of just `string`

Reply:
ahh good point